lib/measures/urban_geometry_creation/measure.rb

```diff
@@ -100,7 +99,19 @@ def run(model, runner, user_arguments)
     model.getBuildingStorys.each { |story| stories << story }
     stories.sort! { |x, y| x.nominalZCoordinate.to_s.to_f <=> y.nominalZCoordinate.to_s.to_f }
 
-    space_types = URBANopt::GeoJSON::Helper.create_space_types(stories, model, runner)
```

Comment:
@tanushree04 I missed this earlier, but this function already exists as well, https://github.com/urbanopt/urbanopt-geojson-gem/blob/c3302d17ab3b53bb535fc10c0db3d10fd52b4174/lib/urbanopt/geojson/helper.rb#L138.

Can you update this to use that method?